toolkit/scripts/setuplkgtoolchain.sh

```diff
@@ -23,39 +28,97 @@ get_lkg() {
 }
 
 check_for_modified_manifests() {
+    auto=$1
+    force=$2
     if git status --porcelain | grep -qP "(pkggen_core|toolchain)_$(uname -p).txt"; then
-        echo "Local modifications to 'pkggen_core_$(uname -p).txt' or 'toolchain_$(uname -p).txt' detected."
-        echo -e "\nNOTE: Changes to manifests were detected, and these will be overwritten. Hit Ctrl+C within 10 seconds to cancel...\n"
-        sleep 10s
+        if $force; then
+            echo "Forcing overwrite of local changes to 'pkggen_core_$(uname -p).txt' or 'toolchain_$(uname -p).txt'" 1>&2
+        else
+            if $auto; then
+                echo "Local modifications to 'pkggen_core_$(uname -p).txt' or 'toolchain_$(uname -p).txt' detected, will not auto update without --force" 1>&2
+                exit 1
+            else
+                echo "Local modifications to 'pkggen_core_$(uname -p).txt' or 'toolchain_$(uname -p).txt' detected." 1>&2
+                echo -e "\nNOTE: Changes to manifests were detected, and these will be overwritten. Hit Ctrl+C within 10 seconds to cancel...\n" 1>&2
+                sleep 10s
+            fi
+        fi
```

Comment:
Idea to reduce indentation and one `fi`:
```suggestion
        elif $auto; then
            echo "Local modifications to 'pkggen_core_$(uname -p).txt' or 'toolchain_$(uname -p).txt' detected, will not auto update without --force" 1>&2
            exit 1
        else
            echo "Local modifications to 'pkggen_core_$(uname -p).txt' or 'toolchain_$(uname -p).txt' detected." 1>&2
            echo -e "\nNOTE: Changes to manifests were detected, and these will be overwritten. Hit Ctrl+C within 10 seconds to cancel...\n" 1>&2
            sleep 10s
        fi
```